luigi/server.py

```diff
@@ -234,7 +234,7 @@ def _init_api(scheduler, responder=None, api_port=None, address=None, unix_socke
     if unix_socket is not None:
         api_sockets = [tornado.netutil.bind_unix_socket(unix_socket)]
     else:
-        api_sockets = tornado.netutil.bind_sockets(api_port, address=address)
```

Comment:
This can fail:
    if reuse_port and not hasattr(socket, "SO_REUSEPORT"):
        raise ValueError("the platform doesn't support SO_REUSEPORT")